static/index.html

```diff
@@ -7,13 +7,14 @@
 		<!-- having trouble getting the right path on both umbrel and local.. -->
 		<link rel="stylesheet" href="/pj/static/style.css">
 		<link rel="stylesheet" href="style.css">
+		<link rel="icon" type="image/x-icon" href="/pj/static/favicon.ico">
 		<title>nolooking // Lightning PayJoin</title>
 	</head>
 	<body dark-mode>
 		<br>
 		<br>
 		<header class="center">
-			<h1>👻 &nbsp;Lightning PayJoin&nbsp; 👻</h1>
+			<a href="https://nolooking.chaincase.app/" target="_blank" style="text-decoration:none"><h1>👻 &nbsp;Lightning PayJoin&nbsp; 👻</h1></a>
```

Comment:
Reading this makes me realize "👻 &nbsp;Lightning PayJoin&nbsp; 👻" is messy. Should be <span class="h1">👻 &nbsp;<h1>Lightning PayJoin</h1>&nbsp; 👻</span> for proper semantics with the `--h1` css variables, but it's past halloween anyway to get rid of this.

Reply:
Couldn't manage to make this change without getting them onto new lines